Platform/QemuBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -166,7 +166,9 @@ VOID BoardDetection (
   } else {
     BoardId = 1;
   }
-
+  if (FeaturePcdGet (PcdSimicsEnabled)) {

```

Comment:
Let us detect at runtime.   
You can do something like below:
IoWrite32 (0xCF8, PCI_TO_CF8_ADDRESS(PCI_CF8_LIB_ADDRESS (0, 0, 0, PCI_VENDOR_ID_OFFSET)));
VidDid = IoRead32 (0xCFC);
if (VidDid == X58_ICH10_HOST_BRIDGE_VID_DID) {
    // Simics QSP
    BoardId = 2;
  } else {
    ...
  }

Reply:
Done, thanks for the guide!